src/main/fc/fc_msp.c

```diff
@@ -110,12 +112,13 @@
 #endif
 
 #define STATIC_ASSERT(condition, name) \
-    typedef char assert_failed_ ## name [(condition) ? 1 : -1 ]
+    typedef char assert_failed_ ## name [(condition) ? 1 : -1 ] __attribute__((unused))
```

Comment:
I know, I'll fix it in separate PR (removing BUILD_BUG_ON,  STATIC_ASSERT can report error message)